ironfish/src/rpc/routes/wallet/getAccountTransactionDescriptions.ts

```diff
@@ -0,0 +1,130 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as yup from 'yup'
+import { UnsignedTransaction } from '../../../primitives/unsignedTransaction'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+import { AssertHasRpcContext } from '../rpcContext'
+import { getAccount } from './utils'
+
+export type GetAccountTransactionDescriptionsRequest = {
+  unsignedTransaction: string
+  account?: string
+}
+
+export type GetAccountTransactionDescriptionsResponse = {
+  account: string
+  mints: {
+    assetId: string
+    value: string
+    transferOwnershipTo?: string
+  }[]
+  burns: {
+    assetId: string
+    value: string
+  }[]
+  receivedNotes: {
+    assetId: string
+    memo: string
+    owner: string
+    sender: string
+    value: string
+  }[]
+  sentNotes: {
+    assetId: string
+    memo: string
+    owner: string
+    sender: string
+    value: string
+  }[]
```

Comment:
would it be possible to reuse one or more of our existing RPC objects like this https://ironfish.network/developers/documentation/rpc/objects/rpcWalletTransaction